app/views/case_information/_errors.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
+  <h2 class="govuk-error-summary__title" id="error-summary-title">
+    There is a problem
+  </h2>
+  <div class="govuk-error-summary__body">
+    <ul class="govuk-list govuk-error-summary__list">
+      <% if errors[:welsh_offender] %>
+        <li><a href="#case_information_welsh_offender_Yes"><%= errors[:welsh_offender].first %></a></li>
+      <% end %>
+
+      <% if errors[:case_allocation] %>
+        <li><a href="#case_information_case_allocation_NPS"><%= errors[:case_allocation].first %></a></li>
+      <% end %>
+
+      <% if errors[:tier] %>
+        <li><a href="#case_information_tier_A"><%= errors[:tier].first %></a></li>
+      <% end %>
```

Comment:
Rather than manually do this for each form, it might be better to make use of https://github.com/ministryofjustice/offender-management-allocation-manager/blob/master/app/views/shared/_field_errors.html.erb which should handle the errors for a form, and provide links etc

Reply:
I never saw this 🤕 I could have saved so much time. I'll make the changes

Reply:
Is it safe to remove this file from the PR now?